containers/ecr-viewer/src/app/api/save-fhir-data/save-fhir-data-service.ts

```diff
@@ -171,355 +173,208 @@ const saveFhirMetadata = async (
+            encounter_start_date: asDate(metadata.encounter_start_date),
+            encounter_end_date: asDate(metadata.encounter_end_date),
+            reason_for_visit: metadata.reason_for_visit,
+            active_problems: metadata.active_problems,
+          })
+          .execute();
+        if (metadata.patient_addresses) {
+          for (const address of metadata.patient_addresses) {
+            const patient_address_uuid = randomUUID();
+            await trx
+              .insertInto("patient_address")
+              .values({
+                uuid: patient_address_uuid,
+                ...address,
+                period_start: asDate(address.period_start),
+                period_end: asDate(address.period_end),
+                eicr_id: ecrId,
+              })
+              .execute();
+          }
         }
-      }
-
-      if (metadata.rr) {
-        // Loop through each condition/rule object in rr array
-        for (const rrItem of metadata.rr) {
-          const rr_conditions_uuid = randomUUID();
-          const rrConditionsInsertRequest = new sql.Request(transaction);
-
-          // Insert condition into ecr_rr_conditions
-          await rrConditionsInsertRequest
-            .input("UUID", sql.VarChar(200), rr_conditions_uuid)
-            .input("eICR_ID", sql.VarChar(200), ecrId)
-            .input("condition", sql.VarChar(sql.MAX), rrItem.condition)
-            .query(
-              "INSERT INTO ecr_viewer.ecr_rr_conditions VALUES (@UUID, @eICR_ID, @condition)",
-            );
-
-          // Loop through the rule summaries array
-          if (rrItem.rule_summaries && rrItem.rule_summaries.length > 0) {
-            for (const summary of rrItem.rule_summaries) {
-              const ruleSummaryInsertRequest = new sql.Request(transaction);
-
-              // Insert each rule summary with reference to the condition
-              await ruleSummaryInsertRequest
-                .input("UUID", sql.VarChar(200), randomUUID())
-                .input(
-                  "ECR_RR_CONDITIONS_ID",
-                  sql.VarChar(200),
-                  rr_conditions_uuid,
-                )
-                .input("rule_summary", sql.VarChar(sql.MAX), summary.summary)
-                .query(
-                  "INSERT INTO ecr_viewer.ecr_rr_rule_summaries VALUES (@UUID, @ECR_RR_CONDITIONS_ID, @rule_summary)",
-                );
-            }
+        if (metadata.labs) {
+          for (const lab of metadata.labs) {
+            // some fields need renaming
+            const {
+              test_result_ref_range_low: test_result_reference_range_low_value,
+              test_result_ref_range_high:
+                test_result_reference_range_high_value,
+              test_result_ref_range_low_units:
+                test_result_reference_range_low_units,
+              test_result_ref_range_high_units:
+                test_result_reference_range_high_units,
```

Comment:
These got renamed in all places but not in `extended.sql`. Was renaming this on purporse?

Reply:
we're renaming from the bundle name to the sql name. In a follow on we could update the message parser to match our sql and skip the rename, but that felt a smidge too far for this PR